src/scales/linearScale.ts

```diff
@@ -17,6 +16,8 @@ export module Scales {
     constructor(scale: D3.Scale.LinearScale);
     constructor(scale?: any) {
       super(scale == null ? d3.scale.linear() : scale);
+      this.autoMin(-Infinity);
```

Comment:
This name does not tell me anything. I believe from our previous discussions it refers to the feature where the autoDomain is bounded by some min or max value. If that is the case, call it `autoDomainMin`. or even better `autoDomainMinValue`.